smtk/mesh/testing/cxx/UnitTestCellTypes.cxx

```diff
@@ -13,7 +13,7 @@
 
 #include "smtk/mesh/testing/cxx/helpers.h"
 
-#ifndef _MSC_VER
+#ifdef __GNUC__
```

Comment:
OK. (Not necessary for this PR, but there are other places that then need to be changed.)